rules/frr.mk

```diff
@@ -0,0 +1,9 @@
+# FRRouting (frr) package
+
+FRR_VERSION = 3.1
+export FRR_VERSION
+
+FRR = frr_$(FRR_VERSION)-dev_amd64.deb
```

Comment:
I was originally pulling from dev/master branch, but i just changed to 'stable', so just got rid of 'dev' string.